lib/bolt/config.rb

```diff
@@ -107,17 +102,37 @@ def self.default
 
     def self.from_boltdir(boltdir, overrides = {})
       data = Bolt::Util.read_config_file(nil, [boltdir.config_file], 'config') || {}
-      new(boltdir, data, overrides)
+      new(boltdir, data, overrides, load_defaults)
     end
 
     def self.from_file(configfile, overrides = {})
       boltdir = Bolt::Boltdir.new(Pathname.new(configfile).expand_path.dirname)
       data = Bolt::Util.read_config_file(configfile, [], 'config') || {}
+      new(boltdir, data, overrides, load_defaults)
+    end
+
+    def self.load_defaults
+      # Lazy-load expensive gem code
+      require 'win32/dir' if Bolt::Util.windows?
+
+      system_path = if Bolt::Util.windows?
+                      Pathname.new(File.join(Dir::COMMON_APPDATA, 'PuppetLabs', 'bolt', 'etc', 'bolt.yaml'))
+                    else
+                      Pathname.new(File.join('/etc', 'puppetlabs', 'bolt', 'bolt.yaml'))
+                    end
+      user_path = Pathname.new(File.join('~', '.puppetlabs', 'etc', 'bolt', 'bolt.yaml'))
```

Comment:
I think we might need to expand this path? Data isn't getting loaded from this file locally for me...